manifest.php

```diff
@@ -1,5 +1,13 @@
 <?php
 
+use oat\tao\model\user\TaoRoles;
+use oat\taoQtiItem\scripts\update\Updater;
+use oat\taoQtiItem\install\scripts\setXMLParserConfig;
+use oat\taoQtiItem\scripts\install\ItemEventRegister;
+use oat\taoQtiItem\scripts\install\SetQtiCreatorConfig;
+use oat\taoQtiItem\install\scripts\SetDragAndDropConfig;
+use oat\taoQtiItem\install\scripts\addValidationSettings;
```

Comment:
I moved the imports down to the existing ones while merging `develop` into the branch.
Hope, you don't mind.